careers/application-process/index.html

```diff
@@ -0,0 +1,292 @@
+        <p>
+            Sometimes very similar jobs will be posted simultaneously,
+            with one of the primary differences being who is eligible to apply.
+            This difference is based on the applicant’s employment history
+            or other federally recognized status.
+            For example, some jobs are limited to current and former federal employees only.
+            In general, if you see two similar jobs that interest you,
+            we recommend applying through the description
+            that applies most closely to your experience and status.
+            You are, however, welcome to apply to any vacancy at any time.
+        </p>
+    </section>
+
+    <section class="block
+                    block__padded-top
+                    block__border-top
+                    block__flush-bottom">
+        <h3>Federal Ethics Rules and CFPB Ethics Regulations</h3>
+        <p>
+            We hold ourselves to an exemplary standard of integrity in all that we do.
+            Should you join us, you will be subject to federal government-wide ethics rules
+            as well as the additional ethics regulations specific to Bureau employees.
+        </p>
+        <p>
+            CFPB Ethics Regulations will prohibit you and your spouse and minor children
+            from holding financial interests in entities supervised by the CFPB;
+            restrict your ability to work on matters involving certain of your creditors,
+            and prevent you from seeking credit from a lender
+            if you are working on a Bureau matter involving that lender.
+            You may review the CFPB Ethics Regulations here.
+        </p>
+        <p>
+            You can reach to our Ethics team with questions at
+            <a href="mailto:EthicsHelp@cfpb.gov">EthicsHelp@cfpb.gov</a>.
+        </p>
+    </section>
+
+    <section class="block
+                    block__padded-top
+                    block__border-top
+                    block__flush-bottom">
+        <h3>Equal Employment Opportunity</h3>
+        <p>
+            The CFPB is an equal opportunity employer
+            and seeks to create and maintain a vibrant and diverse workforce.
+            Women, minorities, veterans, and people with disabilities are encouraged to apply.
+        </p>
+        <p>
+            If you have questions, concerns, or complaints,
+            you should contact our Office of Civil Rights.
+            See our Equal Employment Opportunity page for more.
+        </p>
+    </section>
+
+    <div class="block
+                block__padded-top
+                block__border-top
+                content-l">
+        <section class="content-l_col content-l_col-1-2">
+            {% include "_template-working-at-cfpb.html" %}
+        </section>
+        <section class="content-l_col content-l_col-1-2">
+            {% include "_template-job-app-process.html" %}
```

Comment:
This is the job-app-process page. This block should probably show something else.